package.json

```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "eslint": "^7.0.0",
-    "eslint-config-silverwind": "^13.3.2",
+    "eslint-config-silverwind": "^13.3.3",
```

Comment:
either one PR not both bumping the dev deps, thanks.

Reply:
OK, wasn't sure to include the other changes in this PR too because there is a conflict between them but you're right, it shouldn't be in both PRs. I removed the shared commits.

Reply:
Thanks,

It would be nice to include if possible screenshots of before and after with fixes on both PR's

Ill need tot test and see how it is, but I suspect it will be OK.